README.de.md

```
@@ -20,63 +20,66 @@
+- **Verbesserungen gegenüber der Hersteller-Firmware.** Umfasst WebUI-Verbesserungen, Linux-OS-Updates, Stabilitäts-/Performance-Fixes und neue Funktionen, die upstream noch nicht oder ggf. nie vorhanden sind.
 
 [mehr...](https://github.com/OpenCCU/OpenCCU/wiki/Einleitung#features)
 
-## :computer: Voraussetzungen
+## :computer: Systemvoraussetzungen
 
-OpenCCU lässt sich auf folgender, kommerziell erhältlicher Kompletthardware (Rechner + HomeMatic-Kommunikationsmodul) installieren:
+OpenCCU kann auf Hersteller-CCU-Hardware, gängigen SBCs und x86-Systemen installiert oder virtuell betrieben werden:
 
-* [CCU3](https://www.homematic-ip.com/en/products/detail/smart-home-central-control-unit-ccu3.html), [ELV-Charly](https://www.elv.de/elv-smart-home-zentrale-charly-starter-set-bausatz.html)
+**Hardware**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)